modules/core/test/unit/bitgo.ts

```diff
@@ -452,6 +452,29 @@ describe('BitGo Prototype Methods', function() {
       responseData.isValid.should.equal(true);
     }));
 
+    it('should include request body as part of the hmac', co(function* () {
+      const url = 'https://bitgo.fakeurl';
+      const body = { test: 'test' };
+
+      const fixedUnixTime = 1627374646;
+      const originalDateNow = Date.now;
```

Comment:
instead of overwriting properties of the `Date` global, `useFakeTimers` from `sinon` is preferred.